pytorch_lightning/plugins/plugin_connector.py

```diff
@@ -120,7 +120,7 @@ def _convert_str_to_plugin(self, plugin):
                     f" {plugin} is not a supported lightning custom plugin."
                     " If you're trying to pass a custom plugin, please pass this as an object to"
                     " Trainer(plugins=[MyPlugin()]."
-                    f" Supported plugins as string input: {(e.name for e in LightningCustomPlugins)}."
+                    f" Supported plugins as string input: {([e.name for e in LightningCustomPlugins])}."
```

Comment:
```suggestion
                    f" Supported plugins as string input: {[e.name for e in LightningCustomPlugins]}."
```

Reply:
Thanks! i thought it was a bit strange haha